### ISSUE TITLE
Download latest version for Ubuntu/Linux Mint not possible

### ISSUE THREAD

**Issue by tullin** [QupZilla/qupzilla#2386]
On Qupzilla website clicking "Download 2.1.2' shows button "Ubuntu" on next site. Clicking this button results in advice "sudo apt-get install qupzilla" to install latest version 2.1.2.

Doing installation on this way results in installing version 1.8.9, not 2.1.2! Your repository is out of date!

**Comment by nowrep** [comment]
That's Ubuntu repository that is out of date.

**Comment by tullin** [comment]
Great, do you think about to udate???!!!

**Comment by nowrep** [comment]
You should not use distribution with outdated packages if you don't want outdated packages, simple as that.

**Comment by tullin** [comment]
What do you mean?! I followed your advices on your website. If your repository is out of date how do I use outdated packages?! Your packages are outdated!

So please give me a serious advice but no phrases!

**Comment by nowrep** [comment]
You can try using AppImage (Other Linux category in downloads) or build it yourself.

**Comment by tullin** [comment]
Ok, but remove your out of date advice for installation of latest version for Ubuntu!!

**Comment by tullin** [comment]
Sorry, my mistake!

Old version was in cache. Everything killed (also hidden config!) then it runs new version 2.1.2.

**Comment by cranes-bill** [comment]
> Ok, but remove your out of date advice for installation of latest version for Ubuntu!!

@nowrep, even better to remove _that awesome shiny bright_ Ubuntu. :smiley: 
It might be replaced with Slackware for example.
`upgradepkg --install-new qupzilla-2.1.2-x86_64-1dj.txz`

**Comment by nowrep** [comment]
Why would I remove it? Latest Ubuntu already have latest QupZilla in repositories.

**Comment by cranes-bill** [comment]
Ah, i didn't know that!

**Comment by tullin** [comment]
Sorry, Linux Mint 18.2 doesn't have latest QupZilla in Repositiories.

**Comment by Chiitoo** [comment]
While the download section itself does not advertise 'latest version' anywhere, as far as I can see, it can certainly be confusing (especially since the front-page mentions a specific version).

A simple disclaimer about the possibility of an out-dated version being served by the distributions would probably help a bunch, as probably would a note that the QupZilla project.is not actually packaging QupZilla for them.

**Comment by tullin** [comment]
Yeah, great idea. But better would be to keep repositories current as others can...

**Comment by nowrep** [comment]
> While the download section itself does not advertise 'latest version' anywhere, as far as I can see, it can certainly be confusing (especially since the front-page mentions a specific version).

Linux users should understand difference between installing software from distribution's repository and other means of obtaining software.

> Yeah, great idea. But better would be to keep repositories current as others can...

ditto

**Comment by tullin** [comment]
Linux users should have trust in statements of developer sites!

Main site announces version 2.1.2, link guides to download site, Ubuntu download button to advice to install with 'sudo apt-get install qupzilla'! What the hell is there to missunderstand and what Linux users have to interpret?!

If developers of a web browser want to be succesful in market they shouldn't seek the mistakes at users...

**Comment by tullin** [comment]
It's a pitty! That bad response for such a nice browser... No chance with that in market.

**Comment by potholiday** [comment]
@tullin When I first started with qupzilla I had the same problem. I have Ubuntu14 and when I did `sudo apt-get install qupzilla` it installed an old version of qupzilla. That version was a disaster I cant open gmail or use full screen in youtube and many other problems. There was no latest packaged version of qupzilla availabe in the net so I had to download the AppImage. So far I am satisified with it and the other advantage is you need not to install it in our system.

**Comment by tullin** [comment]
@potholiday Right, I'm doing same and it works great. But developers should point more service...

**Comment by potholiday** [comment]
@tullin qupzilla is an open source software which according to my knowledge dosen' have any monetary motivation. As this is a free and open source sofware we cant demand anything. They are not obliged to give as any service let alone a good service. Believe me I tried and searched many browsers before qupzilla,  this is the only browser which is light weight, fast and uptodate development. Also I got immediate answers to all of my questions which I asked in the ISSUE section whereas many other browser developers dosent even bothered to reply to my questions.

I know they have to keep installination packages up to date but they may have their reasons. As long as they provide an alternative portable and updated software of qupzilla I am happy. After using the portable version I may not in  the future use the installation files.
